dataset_conversion/create_data_splits.py

```diff
@@ -0,0 +1,125 @@
+"""
+Creates train/validation/test splits for a given dataset and saves the output in JSON file. 
+The JSON file can be used to convert the dataset into nnUNet format using the script convert_bids_to_nnunet.py
+
+Usage: 
+1. If you want to include specific sessions and contrasts:
+python create_data_splits.py --path-data <path-to-bids-dataset> --path-out <path-to-output-dir> --split 0.6 0.2 0.2 --include-sessions ses-01 ses-02 --include-contrasts T1w T2w
+
+2. If you want to include all sessions for a specific contrast: (do not specify the --include-sessions argument)
+python create_data_splits.py --path-data <path-to-bids-dataset> --path-out <path-to-output-dir> --split 0.6 0.2 0.2 --include-contrasts T1w T2w
+
+Author: Naga Karthik
+"""
+
+import argparse
+import glob
+import json
+import os
+
+import numpy as np
+import pandas as pd
+from loguru import logger
+from sklearn.model_selection import train_test_split
+from tqdm import tqdm
+import utils_dataset_conversion as utils
+
+parser = argparse.ArgumentParser(description='Code for creating data splits.')
```

Comment:
I would extend the description provided by `create_data_splits.py -h`

```suggestion
parser = argparse.ArgumentParser(description="Creates train/validation/test splits for a given dataset and saves the output in JSON file. The JSON file can be used to convert the dataset into nnUNet format using the script convert_bids_to_nnunet.py")
```